### ISSUE TITLE
Persona: In Scan Mode, 

### ISSUE THREAD

**Issue by aneeshack4** [OfficeDev/office-ui-fabric-react#10850]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:

Bug1 of #7643: Narrator in scan mode doesn't announce the role and State of 'Include Persona Details' checkbox.

### Please provide a reproduction of the issue in a codepen:

Turn on the Narrator and set the scan mode.   1. Open https://uifabric-prod.azurewebsites.net/?example=jhkjhkj#/components/persona in Edge.   2. Navigate to 'Include Persona Details' checkbox using 'Down' arrow.   3. Observe the issue.

#### Actual behavior:

Narrator doesn't announce the role ('Checkbox') and pre-state ('Checked/Unchecked') of contol 'Include Persona Details' using scan mode for navigation.      Observation:    Same issue is repliacted with NVDA(2018.4)

#### Expected behavior:

Narrator in scan mode  should announce the role ('Checkbox') and pre-state ('Checked/Unchecked') of contol 'Include Persona Details'.

**Comment by aneeshack4** [comment]
This no longer repros: 
![image](https://user-images.githubusercontent.com/4161791/66876363-83a4a000-ef7f-11e9-945f-1abd48fe43ed.png)
